src/renderer/widgets/PasswordInputField.cpp

```diff
@@ -423,7 +433,7 @@ void CPasswordInputField::updateColors() {
     col.stateNum  = col.invertNum ? !g_pHyprlock->m_bNumLock : g_pHyprlock->m_bNumLock;
     col.stateCaps = g_pHyprlock->m_bCapsLock;
 
-    if (placeholder.failID.empty()) {
+    if (!placeholder.isFailText) {
```

Comment:
Just keep this checks (from patch above) in mind, maybe there is a way to make them not so ugly xD after work is done.